src/code_80086E70.c

```diff
@@ -1558,23 +1558,23 @@ void func_8008B284(s32 objectIndex) {
 
 void func_8008B3E4(s32 objectIndex) {
     Vec4s *test;
-    struct_80165C18_entry *temp_v0;
+    Objects *temp_v0;
 
-    if (func_80072270(objectIndex, 8) != 0) {
-        temp_v0 = &D_80165C18[objectIndex];
+    if (is_obj_index_flag_unk_054_unactive(objectIndex, 8) != 0) {
```

Comment:
unactive is misspelled:
`is_obj_index_flag_unk_054_unactive`
inactive:
`is_obj_index_flag_unk_054_inactive`